src/adt/test_utils.rs

```diff
@@ -18,6 +18,21 @@ fn gen_bytes<const BYTES_LENGTH: usize>(rng: &mut impl RngCore) -> [u8; BYTES_LE
     bytes
 }
 
+fn u128_to_array<const WORD_LENGTH: usize>(u: u128) -> [u8; WORD_LENGTH] {
+    let mut bytes = [0u8; WORD_LENGTH];
+    bytes[..16].copy_from_slice(&u.to_be_bytes());
+    bytes
```

Comment:
panics if `WORD_LENGTH < 16`.

Reply:
done in 42a347ea9cebd6392499c8933cc7de869946bcea